composer/datasets/c4.py

```diff
@@ -130,7 +130,7 @@ class C4Dataset(IterableDataset):
     HuggingFace's C4 Dataset with streaming backend (See https://huggingface.co/datasets/c4 for more details). The text
     samples are then shuffled, tokenized, and grouped on-the-fly.
 
-    Parameters:
+    Args:
```

Comment:
The reference to `torch.utils.data.IterableDataset` in the "Returns" section of the docstring isn't resolving.